azure/terraform.sh

```diff
@@ -75,6 +75,16 @@ function other_actions() {
   fi
 }
 
+## Apply state don't have -var-file or vars because are inherith from output state
```

Comment:
"The apply state does not require a -var-file or vars because these are inherited from the output state."?

Reply:
done